src/pages/proposals/ProposalList.vue

```diff
@@ -449,9 +452,9 @@ q-page.page-proposals
 
   .row.q-py-md(v-if="$q.screen.gt.md")
     .col-9
-      base-placeholder.q-mr-sm(v-if="!proposals.length && !filteredStagedProposals.length && !filteredProposals.length && !stagedProposals.length" title= "No Proposals" subtitle="Your organization has not created any proposals yet. You can create a new proposal by clicking the button below."
+      base-placeholder.q-mr-sm(v-if="!filteredStagedProposals.length && !filteredProposals.length && !hasProposals" title= "No Proposals" subtitle="Your organization has not created any proposals yet. You can create a new proposal by clicking the button below."
```

Comment:
@evgenibir why do you still need `filteredStagedProposals` ? maybe you can also add them to the computed so that you have clear intetion `v-if="!hasProposals"`

Reply:
also you have double `"!filteredStagedProposals.length && !filteredProposals.length`

Reply:
@arsenijesavic you probably didn’t understand me, we have two widgets, one should be displayed provided that proposals exist (active or staged), but do not fit the filtering conditions, so I left these variables, and the second one is displayed when there are no proposals at all